docs/source/troubleshooting/troubleshooting-getting-started.adoc

```diff
@@ -138,3 +138,18 @@ Downloading https://github.com/minishift/minishift/releases/download/v1.2.0/mini
  65 B / 65 B [===================================================================================================================================================] 100.00% 0s
 Update failed: open /usr/bin/.minishift.new: permission denied
 ----
+
+[[openshift-web-console-doesnot-work-in-older-safari]]
+== OpenShift web console does not work with older versions of Safari
+
+OpenShift web console URL i.e. `minishift console` does not work on older versions of Safari web browser such as version 10.1.2 (12603.3.8).
```

Comment:
Using "i.e." here doesn't really make sense. What we really want to say here is something along the lines of "as accessed by", not that the URL *is* ("id est") `minishift console`.

We should also use "the" before "OpenShift web console URL". :)